src/controllers/resources/resourceController.ts

```
@@ -0,0 +1,334 @@
+
+    logger.info(`got a request to create resource for user: ${user.username}`);
+
+    if(!user.verified_profile){
+      logger.info(`User is not verified user: ${user.username}`);
+      throw new AuthenticationError('User is not verified');
+    }
+
+    if (typeof req.body?.parameters === 'string') {
+      try {
+        req.body.parameters = JSON.parse(req.body.parameters);
+      } catch (e) {
```

ESLint

[error] 'e' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)